tests/perf-test/SidebarUtils.perf-test.ts

```diff
@@ -86,7 +86,7 @@ test('getOrderedReportIDs on 5k reports', async () => {
                 },
             ],
         ]),
-    ) as unknown as OnyxCollection<ReportActions>;
+    ) as unknown as Record<string, ReportAction[]>;
```

Comment:
Where is this `Record<string, ReportAction[]>` structure from?

Reply:
Actually it should be `OnyxCollection<ReportAction[]>` I will change it